src/fontra/core/server.py

```diff
@@ -77,6 +78,9 @@ def setup(self):
         self.httpApp.on_shutdown.append(self.closeProjectManager)
         self._activeWebsockets = set()
 
+        if self.httpPort == 8000:
```

Comment:
Suggest to do this in `__main__.py`, while keeping the helper function in `server.py`.